libbpf-rs/src/btf/mod.rs

```diff
@@ -149,6 +149,17 @@ impl Btf<'static> {
         inner(path.as_ref())
     }
 
+    /// Load the vmlinux btf information from few well-known locations.
+    pub fn load_vmlinux_btf() -> Result<Self> {
```

Comment:
I understand why you opted for this name, but it seems inconsistent with respect to `from_path` in the grand scheme of things. What do you think about `from_vmlinux` or `from_system_vmlinux` instead?

Also, should we add a test for this functionality?

Reply:
> I understand why you opted for this name, but it seems inconsistent with respect to `from_path` in the grand scheme of things. What do you think about `from_vmlinux` or `from_system_vmlinux` instead?
>

`from_vmlinux` is great. 

> Also, should we add a test for this functionality?

will do.